README.md

```diff
@@ -1,2 +1,8 @@
 # t5-commons
-Code required for the Taskforce5 commons project.
+Code required for the Taskforce5 commons project. 
+
+# Packages
+* **common** :  The `t5common` Python package for common operations in the Taskforce5 commons, e.g., interacting with JIRA and creating compute jobs
+* **bilbomd** : Code for connecting bilbomd with Jamo and Jira
+* **dm** : Code and other files for setting up data management with JAMO for T5. This also includes everything necessary for deploying to Kubernetes.
```

Comment:
```suggestion
* **data_management** : Code and other files for setting up data management with JAMO for T5. This also includes everything necessary for deploying to Kubernetes.
```

Update to reflect change: https://github.com/lbl-cbg/t5-commons/pull/37#issuecomment-2688772885